tests/Extensions/Abstractions.UnitTests/EventSourcesEventIdsTests.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.Linq;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Microsoft.Omex.Extensions.Abstractions.UnitTests
+{
+	[TestClass]
+	public class EventSourcesEventIdsTests
+	{
+		[TestMethod]
+		public void EventIdsUnique()
```

Comment:
In OMEX, our unit test name format is Action_WhenX_ShouldY()

Reply:
All unit test renamed, thank you